README.md

```diff
@@ -24,34 +24,48 @@ More detailed requirements are outlined in the [Installing the Islandora Essenti
 
 ### Optional Requirements
 
-If you want to support languages other than English download and enable [String Translation](https://drupal.org/project/i18n), and follow our [guide](https://github.com/Islandora/islandora/wiki/Multilingual-Support) for setting up additional languges.
+If you want to support languages other than English, download and enable [String Translation](https://drupal.org/project/i18n), and follow our [guide](https://github.com/Islandora/islandora/wiki/Multilingual-Support) for setting up additional languages.
 
 ## Installation
 
-Before installing Islandora the XACML policies located [here](https://github.com/Islandora/islandora-xacml-policies) should be copied into the Fedora global XACML policies folder. This will allow "authenticated users" in Drupal to access Fedora API-M functions. It is to be noted that the `permit-upload-to-anonymous-user.xml` and `permit-apim-to-anonymous-user.xml` files do not need to be present unless requirements for anonymous ingesting are present.
+This is one Drupal module in a suite of modules (and stack of related software) which are required for Islandora to function correctly. For full installation instructions please see the [documentation in the DuraSpace Wiki](https://wiki.duraspace.org/display/ISLANDORA/Islandora).
 
-You will also have to remove some default policies if you want full functionality as well.
+### Global Fedora XACML policies
+Before installing Islandora, the XACML policies located [here](https://github.com/Islandora/islandora-xacml-policies) should be copied into the Fedora global XACML policies folder. This will allow "authenticated users" in Drupal to access Fedora API-M functions (create, edit, and delete objects in Fedora).
 
-Remove `deny-purge-datastream-if-active-or-inactive.xml` to allow for purging of datastream versions.
+Notes:
+* The `permit-upload-to-anonymous-user.xml` and `permit-apim-to-anonymous-user.xml` policies allow anonymous (unauthenticated)
+users to create objects and datastreams. They do not need to be present if this is not required.
```

Comment:
Could this be made simpler? Its a negation of an afirmation of a negation.
"They do not need to be present if this is not required."
What about. "Remove .. xml and ...xml to disallow anonymous users to create Objects and datastreams". I really can't imagine a scenario in islandora where that is desired... English not my "forte" of course.

Reply:
Because that's only one part of disallowing it. The rest is in Drupal permissions. What about 
"Remove the ...xml and ...xml policies unless you want to allow anonymous (unauthenticated) users to create Islandora objects (not recommended)."

Reply:
Reworded so they both encourage you to delete them, seeing the wiki for more details.